Errors when running with '--write **/*'

:wave: Trying out prettier on a new Rails project, I ran the README suggestion of:

```
bundle exec rbprettier --write '**/*'
```

And I get a lots of `error` output such as:

```
[error] No parser could be inferred for file: .browserslistrc
[error] No parser could be inferred for file: .byebug_history
[error] No parser could be inferred for file: .env
[error] No parser could be inferred for file: .env.test
[error] No parser could be inferred for file: .gitattributes
.github/workflows/ruby.yml
[error] .github/workflows/ruby.yml: Error: Cannot find module './parser-yaml'
[error] Require stack:
[error] - /Users/andyw8/.gem/ruby/2.7.2/gems/prettier-1.5.4/node_modules/prettier/index.js
[error] - /Users/andyw8/.gem/ruby/2.7.2/gems/prettier-1.5.4/node_modules/prettier/bin-prettier.js
[error]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:892:15)
[error]     at Function.Module._load (internal/modules/cjs/loader.js:742:27)
[error]     at Module.require (internal/modules/cjs/loader.js:964:19)
[error]     at require (internal/modules/cjs/helpers.js:88:18)
[error]     at Object.get yaml [as yaml] (/Users/andyw8/.gem/ruby/2.7.2/gems/prettier-1.5.4/node_modules/prettier/index.js:57415:12)
[error]     at resolveParser (/Users/andyw8/.gem/ruby/2.7.2/gems/prettier-1.5.4/node_modules/prettier/index.js:13587:21)
[error]     at normalize$1 (/Users/andyw8/.gem/ruby/2.7.2/gems/prettier-1.5.4/node_modules/prettier/index.js:13695:18)
[error]     at format (/Users/andyw8/.gem/ruby/2.7.2/gems/prettier-1.5.4/node_modules/prettier/index.js:15111:46)
[error]     at Object.formatWithCursor (/Users/andyw8/.gem/ruby/2.7.2/gems/prettier-1.5.4/node_modules/prettier/index.js:57542:12)
[error]     at format (/Users/andyw8/.gem/ruby/2.7.2/gems/prettier-1.5.4/node_modules/prettier/bin-prettier.js:15883:14)
[error] No parser could be inferred for file: .gitignore
```

I'm not sure if this is an oversight in the README, i.e. it should instead be `**/*.rb`, or if there's some other issue happening?

Using v1.5.4 but I tried v1.5.0 and got the same result.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Changed
 
 - [#841](https://github.com/prettier/plugin-ruby/issues/841) - LoganBarnett, kddeisz - Better error messaging for when unix sockets are not supported by netcat.
+- [#844](https://github.com/prettier/plugin-ruby/issues/844) - andyw8, kddeisz - Ensure we ship all of the parsers with the prettier gem so that `rbprettier` can parse other languages as well.
 
 ## [1.5.4] - 2021-03-17
 

--- a/prettier.gemspec
+++ b/prettier.gemspec
@@ -14,15 +14,11 @@ Gem::Specification.new do |spec|
 
   spec.files =
     Dir.chdir(__dir__) do
-      Dir['{{exe,lib,src}/**/*,*.md}'] + %w[
-        bin/console
-        LICENSE
-        node_modules/prettier/bin-prettier.js
-        node_modules/prettier/index.js
-        node_modules/prettier/third-party.js
-        package.json
-        rubocop.yml
-      ]
+      %w[LICENSE bin/console package.json rubocop.yml] +
+        Dir['{{exe,lib,src}/**/*,*.md}'] +
+        Dir[
+          'node_modules/prettier/{index,bin-prettier,third-party,parser-*}.js'
+        ]
     end
 
   spec.bindir = 'exe'

